engines/misc/src/test/java/org/alfresco/transform/misc/transformers/TextToPdfContentTransformerTest.java

```diff
@@ -43,14 +47,19 @@
 import java.util.HashMap;
 import java.util.Map;
 
-import static org.alfresco.transform.common.RequestParamMap.PAGE_LIMIT;
-import static org.alfresco.transform.common.RequestParamMap.SOURCE_ENCODING;
-import static org.junit.jupiter.api.Assertions.assertEquals;
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.apache.pdfbox.pdmodel.font.PDType1Font;
+import org.apache.pdfbox.text.PDFTextStripper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;;
 
 public class TextToPdfContentTransformerTest
 {
     TextToPdfContentTransformer transformer = new TextToPdfContentTransformer();
 
+    private static final String TEXT_WITH_ABREVE = "Gămbardella, Matthew, Corets, Evă";
```

Comment:
I think this (and the next variable) should have an extra underscore in the name:
```suggestion
    private static final String TEXT_WITH_A_BREVE = "Gămbardella, Matthew, Corets, Evă";
```

Reply:
As the error fixed by this PR mentioned the `abreve` word I tended to use it across the code, however, I have changed the `abreve` occurrences to `breve`.

> java.lang.IllegalArgumentException: U+0103 ('abreve')